src/lib/components/linetimeseriechart/linetimeseriechart.component.tsx

```diff
@@ -147,21 +142,33 @@ export type LineChartProps = (
   timeFormat?: 'date-time' | 'date';
   yAxisTitle?: string;
   helpText?: string;
+  renderTooltip: (
```

Comment:
I think it should be optional and default to the fefault one
```suggestion
  renderTooltip?: (
```